index.js

```diff
@@ -9,65 +9,100 @@ const newMember = require("./data/new-member");
  * @param {Object} obj -
  * @return {number} The number of keys in the object
  */
-function numberOfKeys(obj) {}
+function numberOfKeys(obj) {
+  const keys = _.keysIn(yogaClasses[0]);
+  return keys.length;
+}
 
 /**
  * Remove the falsy values in a numbers array and return the sum
  * @param {numbers[]} array - An array of numbers that can also contain some falsy values
  * @return {number} The sum of the numbers in an array
  */
-function sumNumbers(array) {}
+function sumNumbers(array) {
```

Comment:
Replace 'simpleArray' with the 'array' parameter to use the function's input correctly. Also, remove the unnecessary parentheses around 'sumTruthy' in the return statement.